### PR TITLE
Use pipeline in codegen_test

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

File: hack/generator/pkg/astmodel/function_test.go [hunk 1]
```diff
@@ -30,11 +30,11 @@ func (fake *FakeFunction) AsFunc(codeGenerationContext *CodeGenerationContext, r
 }
```
Comment (matthchr) [inline hack/generator/pkg/astmodel/function_test.go:31]: I think we missed a make all (`go fmt`) in some other PR which is where this came from

File: hack/generator/pkg/codegen/code_generator.go [hunk 2]
```diff
@@ -145,23 +157,37 @@ func (loader *cancellableJSONLoader) LoaderFactory() gojsonschema.JSONLoaderFact
...
+
+	schema, err := sl.Compile(loader)
+	if err != nil {
+		return nil, errors.Wrapf(err, "error loading schema from %q", source)
+	}
+
+	return schema, nil
+}
+
+func loadSchemaIntoTypes(
```
Comment (Porges) [inline hack/generator/pkg/codegen/code_generator.go:177]: while you’re here do you want to move this PipelineStage generator function and all its dependencies into another file? it has a lot of cruft around it for cancellation

Comment (matthchr) [reply hack/generator/pkg/codegen/code_generator.go:177]: sure

Comment (Porges) [reply hack/generator/pkg/codegen/code_generator.go]: @matthchr feels nicer 👍

**Pull request merged**